src/platforms/node/common/configuration/integrations/default-integrations.mdx

```diff
@@ -100,6 +100,21 @@ Available options:
 }
 ```
 
+### ContextLines
+
+_Import name: `Sentry.Integrations.ContextLines`_
```

Comment:
We'll have to add a note, new in version X.

```suggestion
_Import name: `Sentry.Integrations.ContextLines`_

_(Available in version 6.X.X and above)_
```

Reply:
So the JavaScript PR gets merged, add version here and then release together or something like that?

Reply:
Yup exactly!